static/app/components/workflowEngine/gridCell/automationTitleCell.tsx

```diff
@@ -15,6 +15,7 @@ export default function AutomationTitleCell({automation}: Props) {
       name={automation.name}
       link={makeAutomationDetailsPathname(organization.slug, automation.id)}
       systemCreated={!automation.createdBy}
```

Comment:
there shouldn't be any sentry created automations, but some automations are missing the `createdBy` field 🤔 
regardless i think `systemCreated` should always be `false` for automations

Reply:
Ah yes, I'll add that too then!